src/TypeScript/Analyzer/EslintBridgeIssueConverter.cs

```diff
@@ -46,6 +46,17 @@ public IAnalysisIssue Convert(string filePath, Issue issue)
             var ruleDefinitions = rulesProvider.GetDefinitions();
             var ruleDefinition = ruleDefinitions.Single(x => x.EslintKey != null && x.EslintKey.Equals(issue.RuleId, StringComparison.OrdinalIgnoreCase));
             var sonarRuleKey = ruleDefinition.RuleKey;
+            ITextRange textRange = null;
+
+            if(issue.Line != 0)
```

Comment:
This deserves a comment that for the JS/TS analyzer `Line = 0` means the issue is a file-level issue.